data/mods/Aftershock/maps/terrain_groundxeno.json

```
@@ -114,5 +96,309 @@
         { "item": "material_zincite", "count": [ 0, 5 ], "prob": 2 }
       ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_fuma_ice",
+    "name": "fumarolic ice tower",
```

```suggestion
    "name": "ice smoker",
```
The name seems a bit weird `ice smoker` mirrors the name of the fumaroles underwater (`black smoker`).

A fully optional change suggestion.

---

That is true and ice smoker does roll off the tongue easier, I mainly had it as fumarolic ice tower as that is the name used IRL for these things, but I could keep the name if it would help differentiate between the active fumarolic ice tower vs the inactive one

---

Oh nevermind then. Keep the IRL name, thats pretty cool!